Issue with new install

So I decided to start fresh to make sure i did it right and this is the message I am getting when i run crontab
madtaz@OrignalDarkNet:~/Downloads/xteve_lazystream$ docker exec -it xteve_lazystream ./cronjob.sh
2020/08/20 16:15:08 [cronjob.sh] Running scripts...
2020/08/20 16:15:08 [cronjob.sh] Running Lazystream (NHL  via akc)...
error: The following required arguments were not provided:
    <FILE>

USAGE:
    lazystream generate xmltv <FILE> [--start-channel INT] [OPTIONS]

For more information try --help
2020/08/20 16:15:08 [cronjob.sh] Running Lazystream (MLB  via akc)...
error: The following required arguments were not provided:
    <FILE>

USAGE:
    lazystream generate xmltv <FILE> [--start-channel INT] [OPTIONS]

For more information try --help
2020/08/20 16:15:08 [cronjob.sh] Running guide2Go...
sed: /guide2go/"madtaz.yaml": No such file or directory
sed: /guide2go/"madtaz.yaml": No such file or directory
2020/08/20 16:15:08 [G2G  ] Version: 1.1.2
2020/08/20 16:15:08 [ERROR] open /guide2go/"madtaz.yaml: no such file or directory
2020/08/20 16:15:08 [cronjob.sh] Updating xTeVe...
2020/08/20 16:15:08 [cronjob.sh] {
2020/08/20 16:15:08 [cronjob.sh] "status": true
2020/08/20 16:15:08 [cronjob.sh] }{
2020/08/20 16:15:08 [cronjob.sh] "status": true
2020/08/20 16:15:08 [cronjob.sh] }{
2020/08/20 16:15:08 [cronjob.sh] "status": true


This is what my docker fill has

# Env
ENV PUID=1000 \
    PGID=1000 \
    XTEVE_PORT=${XTEVE_PORT} \
    use_xTeveAPI=yes \
    use_lazystream=yes \
    include_nhl=yes\
    include_mlb=yes \
    cdn=akc \ 
    use_guide2go=yes \
    YamlList="madtaz.yaml" \

--- a/root/cronjob.sh
+++ b/root/cronjob.sh
@@ -14,25 +14,39 @@ echo "Running scripts..."
 ### Generate playlist and XML data from Lazystream
 if [ "$use_lazystream" = "yes" ]; then
 
-	args=""
-	if [ -z $quality ]; then args+=" --quality $quality"; fi
-	if [ "$cdn" = "l3c" ]; then args+=" --cdn l3c"; fi
+	args=()
+	if [ ! -z ${quality} ]; then args+=("--quality" "$quality"); fi
+	if [ "$cdn" = "l3c" ]; then args+=("--cdn" "l3c"); fi
 
 	if [ "$include_nhl" = "yes" ]; then
 		echo "Running Lazystream (NHL $quality via $cdn)..."
 		mkdir -p /playlists/lazystream
-		lazystream generate xmltv \
-			--channel-prefix Lazystream:\ NHL \
-			--start-channel 1000 $args \
-			/playlists/lazystream/lazystream-nhl
+
+		nhl_args=()
+
+		nhl_args+=("--channel-prefix")
+		nhl_args+=("Lazystream: NHL")
+		nhl_args+=("--start-channel")
+		nhl_args+=("1000")
+		nhl_args+=("/playlists/lazystream/lazystream-nhl")
+
+		lazystream generate xmltv "${args[@]}" "${nhl_args[@]}"
 	fi
 	if [ "$include_mlb" = "yes" ]; then
 		echo "Running Lazystream (MLB $quality via $cdn)..."
 		mkdir -p /playlists/lazystream
-		lazystream --sport mlb generate xmltv \
-			--channel-prefix Lazystream:\ MLB \
-			--start-channel 2000 $args \
-			/playlists/lazystream/lazystream-mlb
+
+		mlb_args=()
+
+		mlb_args+=("--sport")
+		mlb_args+=("mlb")
+		mlb_args+=("--channel-prefix")
+		mlb_args+=("Lazystream: MLB")
+		mlb_args+=("--start-channel")
+		mlb_args+=("2000")
+		mlb_args+=("/playlists/lazystream/lazystream-mlb")
+
+		lazystream generate xmltv "${args[@]}" "${mlb_args[@]}"
 	fi
 fi
 

